classes/UpgradeSelfCheck.php

```diff
@@ -269,6 +361,33 @@ private function checkModuleVersion()
         return false;
     }
 
+    /**
+     * Check current PHP version is supported.
+     * Returns an URL if unmaintained.
+     * 
+     * @return bool|string
+     */
+    private function checkPhpVersionNeedsUpgrade()
+    {
+        if (PHP_VERSION_ID >= self::RECOMMENDED_PHP_VERSION) {
```

Comment:
use http://php.net/version_compare instead

Reply:
As we deal with integer and not X.X.X strings, is it really useful?

Reply:
Oh, my bad, read `PHP_VERSION` instead of `PHP_VERSION_ID` ><